File: async-scala-client/src/main/scala/io/swagger/client/model/ReceiptTransactionOut.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class ReceiptTransactionOut (
  header: PayRecHeader,
lines: List[PayRecLinesOut],
calculatedTaxSummary: PayRecCalculatedTaxSummaryForService,
processingInfo: ProcessingInfo
)

File: javascript-closure-angular-client/API/Client/ReceiptTransactionOut.js
goog.provide('API.Client.ReceiptTransactionOut');

/**
 * @record
 */
API.Client.ReceiptTransactionOut = function() {}

/**
 * @type {!API.Client.PayRecHeader}
 * @export
 */
API.Client.ReceiptTransactionOut.prototype.header;

/**
 * @type {!Array<!API.Client.PayRecLinesOut>}
 * @export
 */
API.Client.ReceiptTransactionOut.prototype.lines;

/**
 * @type {!API.Client.PayRecCalculatedTaxSummaryForService}
 * @export
 */
API.Client.ReceiptTransactionOut.prototype.calculatedTaxSummary;

/**
 * @type {!API.Client.ProcessingInfo}
 * @export
 */
API.Client.ReceiptTransactionOut.prototype.processingInfo;

